aicsimageio/readers/reader.py

```diff
@@ -474,7 +493,7 @@ def physical_pixel_sizes(self) -> PhysicalPixelSizes:
         We currently do not handle unit attachment to these values. Please see the file
         metadata for unit information.
         """
-        pass
+        return PhysicalPixelSizes(1.0, 1.0, 1.0)
```

Comment:
I might suggest returning 0s or negative values. Then if someone is lazy and hasn't read the docs and gets those values they will immediately know something is up and hopefully they should read more.

Reply:
Thoughts @toloudis?

Reply:
1.0 is the value that best prevents user code from breaking.  If it's a significant enough situation to merit causing division by 0 or messing up visualization code, then by all means make them 0 or negative.  Otherwise just issue a warning or something.